api/api.proto

```diff
@@ -80,16 +80,34 @@ message DeletePasswordResp {
   bool not_found = 1;
 }
 
+// Version holds the version info of components.
+message Version {
+  // Semantic version of the server.
+  string server = 1;
+  // Numeric version of the API.
+  int32 api = 2;
+}
+
+// VersionReq is a request to fetch version info.
+message VersionReq {}
+
+// VersionResp returns version info.
+message VersionResp {
```

Comment:
This can probably just hold the `Version` fields directly. We don't need the `Version` message.